src/pure/runner.ts

```diff
@@ -93,17 +87,15 @@ export class TestRunner {
       args.push("-t", testNamePattern);
     }
 
-    const workspacePath = this.workspacePath;
+    const workspacePath = sanitizeFilePath(this.workspacePath);
     let error: any;
     let outputs: string[] = [];
     const env = { ...process.env, ...workspaceEnv };
     try {
       // it will throw when test failed or the testing is failed to run
       const child = spawn(command, args, {
         cwd: workspacePath,
-        stdio: ["ignore", "pipe", "pipe"],
-        env,
-        shell: isWindows ? "powershell" : false,
```

Comment:
@zxch3n when I remove these and leave shell as `true` it seems to work well. Do you know where these args come from and why they're there?

Reply:
I think they are used to pass true some custom settings from the settings.json

![image](https://user-images.githubusercontent.com/14359811/165029699-86e92985-fb29-4627-a44a-9c3a23c78208.png)

Reply:
Why remove `stdio` and `env`? stdio is used to get the test output. env for nodeEnv config

Reply:
No problem. Put them back 😀